substrate/primitives/io/src/lib.rs

```diff
@@ -1834,6 +2029,7 @@ pub type SubstrateHostFunctions = (
 	crate::trie::HostFunctions,
 	offchain_index::HostFunctions,
 	transaction_index::HostFunctions,
+	virtualization::HostFunctions,
```

Comment:
```suggestion
```

Also this should not be done for experimental host functions.

Reply:
Done